CHANGES.rst

```diff
@@ -1,3 +1,10 @@
+1.13.2 (2018 February 26)
```

Comment:
are you planning to push a new release right after this is merged?

Reply:
Yep

Reply:
Because currently we have only one wheel on PyPi (py2-py3 wheel), that's built by python3 and misses `futures` dependencies. Which means, it's impossible to install it on py2. :(